lib/rubocop/minitest/assert_offense.rb

```diff
@@ -75,11 +75,13 @@ module Minitest
     module AssertOffense
       private
 
-      def setup
-        cop_name = self.class.to_s.delete_suffix('Test')
-        return unless RuboCop::Cop::Minitest.const_defined?(cop_name)
+      def cop
+        @cop ||= begin
```

Comment:
This is the meat of the change, lazily-loading the cop in order to raise a nice error when the cop is used if the class isn't available.